itg.f90

```diff
@@ -282,12 +282,13 @@ PROGRAM itg
             endtime = TimeStep
             new_gamma = SUM( GlobalGamma(begtime:endtime) ) / navg
             delta_gamma = ABS(new_gamma-old_gamma) / ABS(new_gamma)
+            sigma_gamma = ABS(MAXVAL(GlobalGamma(begtime:endtime) - new_gamma)) / ABS(new_gamma)
```

Comment:
I think you want `maxval(abs(` rather than `abs(maxval` to find the biggest difference?

You could also have `std_gamma = sqrt(sum( (globalgamma(begtime:endtime) - new_gamma)**2)/navg)` to compute the standard deviation as well.